src/plugins/embeddable/public/lib/triggers/triggers.ts

```diff
@@ -53,8 +49,16 @@ export interface RangeSelectTriggerContext<T extends IEmbeddable = IEmbeddable>
   };
 }
 
+export type ChartActionContext<T extends IEmbeddable = IEmbeddable> =
```

Comment:
`ChartActionContext` should it be `ChartTriggerContext` ?

Reply:
Could be either, I'll rename it to `ChartTriggerContext` in a follow up PR.